recipes-support/onboard/onboard_%.bbappend

```diff
@@ -0,0 +1,43 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+
+SRC_URI += " \
+	file://0001-add-xfce-to-autostart-onlyshowin.patch \
+	file://0002-onboard-onhover-seg-fault-fix.patch \
+	file://01-gnome-accessibility \
+	file://NI.colors \
+	file://NI.theme \
+	file://onboard-defaults.conf \
+"
+
+CXXFLAGS += "-Werror=declaration-after-statement"
+
+do_install:append () {
+	install -d ${D}${sysconfdir}/dconf/db/local.d
+	install -d ${D}${sysconfdir}/onboard
+	install -d ${D}${datadir}/onboard/themes
+	install -d ${D}${sysconfdir}/xdg/autostart
+
+	install -m 644 ${WORKDIR}/01-gnome-accessibility ${D}${sysconfdir}/dconf/db/local.d/
+	install -m 644 ${WORKDIR}/onboard-defaults.conf ${D}${sysconfdir}/onboard/
+
+	install -m 644 ${WORKDIR}/NI.colors ${D}${datadir}/onboard/themes/
+	install -m 644 ${WORKDIR}/NI.theme ${D}${datadir}/onboard/themes/
+
+	mv ${D}${PYTHON_SITEPACKAGES_DIR}${sysconfdir}/xdg/autostart/onboard-autostart.desktop ${D}${sysconfdir}/xdg/autostart
```

Comment:
We want this to go in the source .bb file not the .bbappend.

Reply:
Fixed.